soap/src/java/com/zimbra/soap/mail/message/RestoreContactsRequest.java

```diff
@@ -35,6 +39,38 @@
     @XmlAttribute(name = MailConstants.A_CONTACTS_BACKUP_FILE_NAME, required = true)
     private String contactsBackupFileName;
 
+    /**
+     * @zm-api-field-tag resolve
+     * @zm-api-field-description restore resolve action
```

Comment:
Please state which is the default here - that is useful for the HTML SOAP docs.
Also, could you document what the various options mean please. I'm guessing ignore means if a contact with the same ID (what identifies a contact?) already exists, skip this one. modify means merge?? replace is fairly clear, no idea what reset means.  Although updating the enum with the docs makes most sense from a source perspective, currently here would be the best place to ensure the information ends up in the SOAP docs.

Reply:
Updated the description. Please review https://github.com/Zimbra/zm-mailbox/pull/364